src/Parse/ParseClient.php

```diff
@@ -124,6 +124,7 @@
      *
      * @throws Exception
      */
+    //@codingStandardsIgnoreLine
```

Comment:
Nvm I took a look at it myself. Considering that this line is rather long I would suggest breaking it down like `ParseClient::_request`. Then just make sure to remove the ignore above it.